appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.mapper.js

```diff
@@ -55,14 +55,15 @@ export async function interestedPartyCommentsPage(
  * @param {{ addressLine1: string, addressLine2: string, town: string, county: string, postCode: string }} address
  * @param {import('@pins/express').ValidationErrors | undefined} errors
  * @param {string} backPath
+ * @param {string} operationType
```

Comment:
Would it be worth defining the possible values `operationType` can have here rather than just a string?

Reply:
Will fix forward in my currently overlapping bug ticket